### PR TITLE
Consolidate Kafka topic list

### PR DESCRIPTION
**Pull request by yoomlam**
## What was the problem?

The Kafka topic list is repeated in multiple places:
- [KafkaConsumer.java](https://github.com/department-of-veterans-affairs/abd-vro/pull/1958/files/9c04495b09d5d8f207c3f238231a992c6f3c2147#diff-4dff030a6e0e98a6f993911f7eb0cde870521d2a6595adb11444b96233b323e3)
- application.yaml

Related comments:
- https://github.com/department-of-veterans-affairs/abd-vro/pull/1957/files/c8ce57c7cb94f2369b8fb2598ade2a1a57fc3b5b..5da31725713f61415705caa2c5e83bcdf0fa85b7#r1302019458
- https://github.com/department-of-veterans-affairs/abd-vro/pull/1869#discussion_r1302026795

## How does this fix it?

Consolidate Kafka topic list to be only in application.yaml

## How to test this PR
[CI: BIE Kafka End-2-End Test](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/5943912048)

### CODE REVIEW

File: svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java [hunk 1]
```diff
@@ -19,4 +21,20 @@ public class BieProperties {
...
    */
   private Map<String, String> kafkaTopicToAmqpExchangeMap;
+
+  @Value("${kafka.topic.prefix}")
+  String topicPrefix;
+
+  @Getter private Map<String, String> topicToExchangeMap;
+
+  @PostConstruct
+  public void addTopicPrefix() {
```
Comment (yoomlam) [inline svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java:31]: Could rename to
```suggestion
  public void addPrefixToTopicNames() {
```

File: svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java [hunk 2]
```diff
@@ -19,4 +21,20 @@ public class BieProperties {
    * separated by a colon ":" character.
    */
   private Map<String, String> kafkaTopicToAmqpExchangeMap;
+
+  @Value("${kafka.topic.prefix}")
```
[the remark this answers is not in the document]
Comment (yoomlam) [reply svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java]: Implemented

File: svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java (hunk 1 above)
Comment (yoomlam) [reply svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/BieProperties.java]: Done.

**Review by chengjie8 — APPROVED** [review]
Thanks for adding a cleaner approach for the topic prefix. Like it!

File: svc-bie-kafka/src/test/java/gov/va/vro/services/bie/config/KafkaConsumerTest.java [hunk 3]
```diff
@@ -1,24 +1,53 @@
...
+    topicToExchange.put(
+        "CONTENTION_BIE_CONTENTION_ASSOCIATED_TO_CLAIM_V02", "bie-events-contention-associated");
+    topicToExchange.put("CONTENTION_BIE_CONTENTION_UPDATED_V02", "bie-events-contention-updated");
+
+    bieProperties.kakfaTopicPrefix = "TST_";
+    bieProperties.addPrefixToTopicNames();
+  }
+
+  //  @Value("#{bieProperties.topicNames(${kafka.topic.prefix})}")
+  //  private String topicNames;
```
Comment (tejans24) [inline svc-bie-kafka/src/test/java/gov/va/vro/services/bie/config/KafkaConsumerTest.java:41]: commented code here

**Review by tejans24 — APPROVED** [review]
LGTM

File: svc-bie-kafka/src/test/java/gov/va/vro/services/bie/config/KafkaConsumerTest.java (hunk 3 above)
Comment (yoomlam) [reply svc-bie-kafka/src/test/java/gov/va/vro/services/bie/config/KafkaConsumerTest.java:41]: Oops! Didn't mean to commit this file.

**Pull request merged**